packages/fulfillment/src/models/service-zone.ts

```diff
@@ -57,6 +70,7 @@ export default class ServiceZone {
     pivotTable: "service_zone_geo_zones",
     joinColumn: "service_zone_id",
     inverseJoinColumn: "geo_zone_id",
+    fixedOrder: true,
   })
   geo_zones = new Collection<GeoZone>(this)
```

Comment:
I see your point, what i was thinking is if you want to remove a geo from a zone then we detach it from the zone. But i can go with that as well i dont have a strong opinion for that. It will also simplify things 👍

Reply:
Let's do it 👍